src/Screens/Liked/liked.css

```diff
@@ -0,0 +1,31 @@
+.Liked {
+    padding: 2rem;
+    display: flex;
+    flex-direction: column;
+    gap: 3rem;
+}
+
+.liked-videos-section {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    gap: 4rem;
+}
+
+.liked-videos-section>*:not(h1) {
+    width: 17.5rem;
+}
+
+.Liked h1 {
```

Comment:
never saw anyone using capital naming in CSS. Just keep a check on whats the usual practice.